src/common/compiler.hpp

```diff
@@ -75,10 +75,13 @@ do {                  \
  */
 #if defined __GNUC__
 #   define BITPIT_DEPRECATED(func) func __attribute__ ((deprecated))
+#   define BITPIT_DEPRECATED_FOR(func, replacement) func __attribute__ ((deprecated(" Use " #replacement)))
 #elif defined __clang__
 #   define BITPIT_DEPRECATED(func) func __attribute__ ((deprecated))
+#   define BITPIT_DEPRECATED(func, replacement) func __attribute__ ((deprecated(" Use " #replacement)))
 #elif defined(_MSC_VER)
     #define BITPIT_DEPRECATED(func) __declspec(deprecated) func
+    #define BITPIT_DEPRECATED(func), replacement __declspec(deprecated(" Use " #replacement)) func
```

Comment:
Same here.

Reply:
Sure, thanks. I put two bugs in three lines of code... 🗡️